common/lib/admin.module.access.php

```diff
@@ -125,7 +125,7 @@ function login (?string $user, ?string $pass)
             return $row;
         }
         // fallback to legacy authentication
-        $pass = filter_var($pass, FILTER_SANITIZE_STRING);
+        $pass = htmlspecialchars($pass, ENT_QUOTES, 'UTF-8');
```

Comment:
`htmlspecialchars` is for output to HTML, not sanitization. This would break logins if an account has certain characters in its password. There's no need to sanitize this at all though, it's just doing a string comparison. Just delete this line.